nginx/nginx.conf

```diff
@@ -345,14 +345,15 @@ http {
             expires -1;
```

Comment:
Can we add a note to the `README` about the requirement of email or having to use the logged email from the console?